src/frontend/tests/e2e/specs/testCreateAccount.js

```diff
@@ -0,0 +1,38 @@
+describe("After loading the app", () => {
```

Comment:
We should also be testing all the different toast messages the user can receive!